build.gradle.kts

```
@@ -44,7 +34,7 @@ subprojects {
 	apply(plugin = "com.github.johnrengelman.shadow")
 
 	lombok {
-		version.set("1.18.16")
+		version.set("1.18.18")
```

Lombok newer version
```suggestion
		version.set("1.18.20")
```